SAT/VariableSelector.cpp

```diff
@@ -177,7 +177,11 @@ for(const TermList* t = _stack.pop(); !t->isEmpty(); t = t->next()){
       }
       RSTAT_MCTR_INC("nscore",level_sum);
       // if we want average, we set divid=true earlier
-      if(divide){ level_sum = (level_sum/count);}//This division will truncate
```

Comment:
Divide-by-zero. I don't think this ever happens, but good to trap it.

Reply:
I don't think we are using the in-house SAT solver anymore (not in the casc strategies...), so this code is not essential...

Reply:
Great - do we intend to retire the in-house solver? If so I can add it to my "unused code" issue.

Reply:
Please, add it to the issue. Let's see whether Giles does not have any plans with it. I am happy with it going.